e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -207,15 +210,15 @@ test.describe('Notebook search tests', () => {
 });
 
 test.describe('Notebook entry tests', () => {
+    // Create Notebook with URL Whitelist
+    let notebook;
+    test.beforeEach(async ({ page }) => {
+        notebook = await startAndAddNotebookWithUrls(page, 'Notebook Entry Tests');
```

Comment:
Can we get rid of this helper function? It's short enough, and I think it's clear enough what's happening if we just put this in the `beforeEach`.